src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,226 @@
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Message_Wall:Sktest",
+            "Message Wall:Sktest - Sktest123 Wiki - Wikia",
+            "Message Wall:Sktest - Sktest123 Wiki - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Thread:2160",
+            "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia",
+            "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Special:Maps",
+            "Maps - Wikia",
+            "Maps - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Template:Welcome",
+            "Template:Welcome - Sktest123 Wiki - Wikia",
+            "Template:Welcome - Sktest123 Wiki - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "MediaWiki:Common.css",
+            "MediaWiki:Common.css - Sktest123 Wiki - Wikia",
+            "MediaWiki:Common.css - Sktest123 Wiki - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "MediaWiki:Edit",
+            "MediaWiki:Edit - Sktest123 Wiki - Wikia",
+            "MediaWiki:Edit - Sktest123 Wiki - Wikia"
+        },
+        // custom title
+        {
+            TEST_WIKI_CUSTOM_TITLE,
+            "WikiDex",
+            "WikiDex, la enciclopedia Pokémon - Wikia",
+            "WikiDex - WikiDex - Wikia"
+        },
+        {
+            TEST_WIKI_CUSTOM_TITLE,
+            "Lista_de_Pokémon",
+            "Lista de Pokémon - WikiDex, la enciclopedia Pokémon - Wikia",
+            "Lista de Pokémon - WikiDex - Wikia"
+        },
+        {
+            TEST_WIKI_CUSTOM_TITLE,
+            "Special:Version",
+            "Versión - WikiDex, la enciclopedia Pokémon - Wikia",
+            "Versión - WikiDex, la enciclopedia Pokémon - Wikia"
+        },
+        {
+            TEST_WIKI_CUSTOM_TITLE,
+            "Categoría:Regiones",
+            "Regiones - WikiDex, la enciclopedia Pokémon - Wikia",
+            "Regiones - Wikia"
+        },
+        {
+            "wikia",
+            "About",
+            "About - Wikis from Wikia - Join the best wiki communities - Wikia",
+            "About - Wikis from Wikia - Join the best wiki communities - Wikia"
+        },
+    };
+  }
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest002() {
+    return new Object[][]{
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Special:Chat",
+            "Sktest123 Wiki: Welcome to the Sktest123 Wiki chat - Wikia"
+        },
+     };
+  }
+
+  /**
+   * Check html title for anon users (the contents of <title>)
+   */
+  @Test(
+      dataProvider = "dataHtmlTitleTest001",
+      groups = {"Seo", "SeoHtmlTitle", "SeoHtmlTitleTest_001"}
+  )
+  public void HtmlTitleTest_001(String wiki, String path, String exp, String expMercury) {
+    if ("CHROMEMOBILEMERCURY".equalsIgnoreCase(Configuration.getBrowser())) {
+      exp = expMercury;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "exp". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)